src/services/ProposalService.ts

```diff
@@ -72,6 +73,15 @@ export class ProposalService {
       coAuthors
     )
 
+    DiscordService.newProposal(
```

Comment:
We could also do this `inBackground` and `await` the async method

Reply:
Now that I think about it, if no one is waiting for all this `DiscordService` methods to return a value, then we could use the `inBackground(async () => { await magic }` inside each one of them, and keep calling them as we are.

WDYT? @andyesp